presto-main/src/main/java/com/facebook/presto/event/QueryMonitor.java

```diff
@@ -328,10 +334,24 @@ private List<OperatorStatistics> createOperatorStatistics(QueryInfo queryInfo)
                         operatorSummary.getPeakTotalMemoryReservation(),
                         operatorSummary.getSpilledDataSize(),
                         Optional.ofNullable(operatorSummary.getInfo()).map(operatorInfoCodec::toJson),
-                        operatorSummary.getRuntimeStats()))
+                        operatorSummary.getRuntimeStats(),
+                        getPlanNodeEstimateOutputSize(operatorSummary.getPlanNodeId(), estimateMap, planIdNodeMap),
+                        estimateMap.containsKey(operatorSummary.getPlanNodeId()) ? estimateMap.get(operatorSummary.getPlanNodeId()).getOutputRowCount() : NaN))
```

Comment:
Add estimated output position counts and output data size in bytes to operator statistics, so that it can be logged later